app/controllers/properties_controller.rb

```diff
@@ -0,0 +1,30 @@
+class PropertiesController < ApplicationController
+
+  def index
+    @properties = Property.all
+  end
+
+  def show
+    @property = Property.find(params[:id])
+  end
+
+  def new
+    @property = Property.new
+  end
+
+  def create
```

Comment:
Does this save property.user_id as the current user id?
I thought you would need to add @property.user = @user or something similar?

Reply:
I think you're right